pyfr/solvers/base/elements.py

```diff
@@ -47,6 +47,9 @@ def __init__(self, basiscls, eles, cfg):
         self.nfpts = basis.nfpts
         self.nfacefpts = basis.nfacefpts
 
+        # Compute metric (Smat, djac on pseudo grid points)
+        self._smat_base()
```

Comment:
I have a commit to remove _gen_pnorm_fpts from the constructor along with the ploc stuff.  I will push later tonight.